src/Microsoft.DotNet.Build.Tasks/PackageFiles/RunnerTemplate.Windows.txt

```diff
@@ -21,6 +21,10 @@ echo Hard linking dependent files...
 :: Format here is: call :copyandcheck Path1 Path2 || GOTO EOF
 [[CopyFilesCommands]]
 
+IF DEFINED FELL_BACK_TO_COPY (
+echo Was unable to make hard links in path, had to fall back to copying dependencies.
```

Comment:
Do we think it is worth giving this message? I kind of don't think it is so I would just exclude it.